shed_tool_conf.xml

```diff
@@ -64104,26 +64104,6 @@
             <version>1.4.0</version>
         </tool>
     </section>
-    <section id="sam_bam" name="SAM/BAM" version="">
```

Comment:
warum löschen wir die? Reicht verstecken nicht einfach?

Reply:
There are exact duplicates of these records in the file. So I thought we could shrink it a tiny bit :)

Reply:
mh, but f7a0d41036c7/sam_to_bam you deleted two times, no?

Reply:
yes, and for each record there is one more I'm keeping (started out with triplicated records)

Reply:
too bad the diff doesn't show the kept one

Reply:
I see ok! Thanks.